packages/discord.js/src/client/actions/MessagePollVoteAdd.js

```diff
@@ -11,11 +14,25 @@ class MessagePollVoteAddAction extends Action {
     const message = this.getMessage(data, channel);
     if (!message) return false;
 
-    const { poll } = message;
+    const includePollPartial = this.client.options.partials.includes(Partials.Poll);
+    const includePollAnswerPartial = this.client.options.partials.includes(Partials.PollAnswer);
+    if (message.partial && (!includePollPartial || !includePollAnswerPartial)) return false;
+
+    if (!message.poll && includePollPartial && includePollAnswerPartial) {
+      message.poll = new Poll(this.client, { ...data, answers: [], partial: true }, message, channel);
+
+      const pollAnswer = new PollAnswer(this.client, data, message.poll);
+
+      message.poll.answers.set(data.answer_id, pollAnswer);
+    }
+
+    const answer = message.poll.answers.get(data.answer_id);
```

Comment:
This is not resolved. This logic is duplicated in the MessagePollVoteRemove action, so move this to a method in the Action class instead.